pkgs/development/python-modules/edge-tts/default.nix

```
@@ -0,0 +1,42 @@
+{
+  fetchFromGitHub,
+  lib,
+  buildPythonPackage,
+  # Dependencies
+  aiohttp,
+  certifi,
+  srt,
+  tabulate,
+  typing-extensions,
+}:
+buildPythonPackage rec {
+  pname = "edge-tts";
+  version = "7.0.0";
```

```suggestion
  version = "7.0.0";
  pyproject = true;

```
This is the preferred way to build Python packages.